src/app/shared/user/user.service.ts

```diff
@@ -45,8 +45,15 @@ export class UserService {
     });
   }
 
-  addUserToWorkflows(userId: number): void {
-    this.alertService.start('Adding user to existing workflows on Dockstore');
+  /**
+   * Add Workflows or GitHub App Tools from a user's organization
+   *
+   * @param userId
+   * @param isWorkflow  Display 'workflows' or 'tools' in the alert message
+   */
+  addUserToWorkflows(userId: number, isWorkflow: boolean): void {
+    // Should use 'tools' when adding app tools from myTools page and 'workflows' when adding from myWorkflows page
+    this.alertService.start('Adding user to existing ' + (isWorkflow ? 'workflows' : 'tools') + ' on Dockstore');
```

Comment:
Two contradictory suggestions:

1. instead of passing a boolean, pass the value or enum to display, perhaps using entry type metadata from #1732. For example, if we ever add this feature for notebooks or some other entity, we don't have to update this code, we just pass in the new display value/enum as a parameter.
2. The web service endpoint discovers both app tools and workflows, so maybe we shouldn't have a type-specific message anyway. Maybe it should just say "Adding user to existing entries on Dockstore", or "Adding user to existing tools and workflows on Dockstore". If we do that, do we also update the menu choices to say "Discover Existing Dockstore Entries"? 

My vote is:

* Keep the menu choice labeling the same.
* Change the success message to "Adding user to existing workflows and tools on Dockstore".

Reply:
> My vote is:
> 
> * Keep the menu choice labeling the same.
> * Change the success message to "Adding user to existing workflows and tools on Dockstore".

Mm I see, I agree.  I'll keep the labels the same and change the success message to both tools and workflows